sheetstorm/management/commands/data_load.py

```diff
@@ -0,0 +1,83 @@
+import logging
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from users.factories import UserFactory
+
+User = get_user_model()
+
+user_type_options = {"admin": "admin", "manager": "manager", "employee": "employee"}
+
+
+class Command(BaseCommand):
+    help = "Create initial sample data for SheetStorm application testing."
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            "-a", "--admin", type=int, help="Indicates the maximum number of admins to be in the database"
+        )
+        parser.add_argument(
+            "-e", "--employee", type=int, help="Indicates the maximum number of employees to be in the database"
+        )
+        parser.add_argument(
+            "-m", "--manager", type=int, help="Indicates the maximum number of managers to be in the database"
+        )
+
+    @staticmethod
+    def add_user(user_type, number_of_users_to_create):
+        if user_type == user_type_options["admin"]:
+            factory_parameters = {"user_type": User.UserType.ADMIN.name, "is_staff": True}
+            password = "admin1234"
+
+        if user_type == user_type_options["employee"]:
+            factory_parameters = {"user_type": User.UserType.EMPLOYEE.name}
+            password = "passwduser"
+
+        if user_type == user_type_options["manager"]:
+            factory_parameters = {"user_type": User.UserType.MANAGER.name}
+            password = "passwduser"
+
+        for new_user in range(number_of_users_to_create):
+            user = User.objects.filter(email=f"user.{user_type}{new_user}@codepoets.it").first()
+            if user is None:
+                user = UserFactory(**factory_parameters, email=f"user.{user_type}{new_user}@codepoets.it")
+                user.set_password(password)
+                user.save()
+                logging.info(f"Add {user_type} {new_user+1}")
+
+    @staticmethod
+    def add_superuser():
+        super_admin = User.objects.filter(email=f"admin@codepoets.it").first()
+        if super_admin is None:
+            super_admin = UserFactory(
+                email=f"admin@codepoets.it",
```

Comment:
Use regular **string** instead of the **f-string**, FLAKE8 is still issuing warnings about it

Reply:
done